prove_stuffs/prove.h

```diff
@@ -36,14 +36,8 @@ char *strncpy(char *dest, const char *src, size_t n)  {
 
   #include "../prove_stuffs/prove_helpers.h"
 #else
-  #ifdef PROVERS_CBMC
-    #include <assert.h>
-    #define p_assert(x) assert(x)
-    #define p_assume(x) __CPROVER_assume(x)
-  #else
-    #define p_assert(x)
-    #define p_assume(x)
-  #endif
+  #define p_assert(x)
```

Comment:
LGTM for me but we restrict the user to only use SH for custom assertions. It might be useful to add a custom header to handle CBMC separately if they still want to use it for custom assertions (e.g. for debugging purposes or any other reason)

Reply:
Yes, this would solve the problem